<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowSerial</class>
 <widget class="QMainWindow" name="windowSerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>791</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,3">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="10,10,1">
       <item>
        <widget class="QGroupBox" name="groupBoxComSet">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>260</height>
          </size>
         </property>
         <property name="title">
          <string>串口设置</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>181</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QComboBox" name="comboBoxPorts"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonUpdatePorts">
               <property name="text">
                <string>刷新</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="0">
              <widget class="QLabel" name="labelData">
               <property name="text">
                <string>数据位</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBoxData">
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelStop">
               <property name="text">
                <string>停止位</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBoxComBaud">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>1200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>38400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>57600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>74880</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>115200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>230400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>460800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>576000</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="comboBoxCheck">
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Even</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Odd</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mark</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Space</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="comboBoxStop">
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="labelCheck">
               <property name="text">
                <string>校验位</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelBaud">
               <property name="text">
                <string>波特率</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonTurnOnOff">
             <property name="text">
              <string>打开串口</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonClearAll">
             <property name="text">
              <string>清空数据</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>待放区域</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>151</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;这块区域还没想好要放什么&lt;/p&gt;&lt;p&gt;以后有新的需求再使用吧&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 20pt &quot;楷体&quot;;
	color:rgb(186, 186, 186);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
}
</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;刘禹惜出品&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="10,2,1,6,1">
       <item row="0" column="0" rowspan="2" colspan="2">
        <widget class="QGroupBox" name="groupBoxRev">
         <property name="title">
          <string>接收区</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="TextRev">
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="plainText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>多行发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxPeriodicSend">
            <property name="text">
             <string>周期发送</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelMS">
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;华文新魏&quot;;
color: rgb(74, 74, 74);</string>
            </property>
            <property name="text">
             <string>发送：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sendByteNum">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>128</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 12pt &quot;华文新魏&quot;;
color: rgb(74, 74, 74);</string>
            </property>
            <property name="text">
             <string>接收：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="receiveByteNum">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">font: 15pt &quot;华文新魏&quot; ;
color: rgb(117, 117, 117);</string>
            </property>
            <property name="text">
             <string>祝你天天开心</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBoxSend">
         <property name="title">
          <string>发送区</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" rowspan="2" colspan="2">
           <widget class="QPlainTextEdit" name="TextSend">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonSend">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
